app/controllers/workshops_controller.rb

```diff
@@ -32,4 +33,18 @@ def set_workshop
   def role_params
     params[:role]
   end
+
+  def find_attending_invitation(workshop, user)
+    WorkshopInvitation.find_by(workshop: workshop, member: user, attending: true)
+  end
+
+  def find_or_create_invitation(workshop, user, role)
+    WorkshopInvitation.find_or_create_by(workshop: workshop,
+                                         member: user,
+                                         role: role)
+  end
+
+  def user_attending_or_waitinglisted?(workshop, user)
```

Comment:
@despo can we rename this method to `user_attending_or_waitlisted?`? Waitinglisted sounds a listen awkward.

Reply:
Ok!